docs/configuration.md

```diff
@@ -330,13 +330,13 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.python.profile</code></td>
   <td>false</td>
   <td>
-    Enable profiling in Python worker, the profile result will show up by `sc.show_profiles()`,
+    Enable profiling in Python worker, the profile result will show up by <code>sc.show_profiles()<code>,
```

Comment:
Sorry for spamming~

On Sat, Nov 14, 2015, 02:58 Sean Owen <notifications@github.com> wrote:

> In docs/configuration.md
> <https://github.com/apache/spark/pull/8795#discussion_r44857336>:
>
> > @@ -330,13 +330,13 @@ Apart from these, the following properties are also available, and may be useful
> >    <td><code>spark.python.profile</code></td>
> >    <td>false</td>
> >    <td>
> > -    Enable profiling in Python worker, the profile result will show up by `sc.show_profiles()`,
> > +    Enable profiling in Python worker, the profile result will show up by <code>sc.show_profiles()<code>,
>
> Yes, you already opened a PR. I don't think it's worth 'dittoing' 5 lines
> and sending 5 emails?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/8795/files#r44857336>.
>